server/processors/src/test/java/org/hawkular/apm/server/processor/tracecompletiontime/TraceCompletionInformationInitiatorTest.java

```diff
@@ -95,6 +95,29 @@ public void testProcessSingleInitialFragmentConsumer() throws RetryAttemptExcept
     }
 
     @Test
+    public void testProcessSingleInitialFragmentProducer() throws RetryAttemptException {
+        Trace trace = new Trace();
+        trace.setTraceId("traceId");
+        trace.setFragmentId(trace.getTraceId());
+
+        Producer p = new Producer();
+        p.setUri("uri");
+        p.addInteractionCorrelationId("pid");
+
+        trace.getNodes().add(p);
+
+        TraceCompletionInformationInitiator initiator = new TraceCompletionInformationInitiator();
+
+        TraceCompletionInformation ci = initiator.processOneToOne(null, trace);
+
+        assertNotNull(ci);
```

Comment:
Its redundant with knowledge of the current implementation - but in terms of the API the `processOneToOne` method does not guarantee to return a value.